provisioning/provisioning-service-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/provisioning/service/configs/EnrollmentGroupTest.java

```diff
@@ -1778,4 +1779,35 @@ public void getIotHubsWorks(final @Mocked SymmetricKeyAttestation mockedAttestat
         //assert
         assertEquals(expectedIotHubs, actualIotHubs);
     }
+
+    /* SRS_ENROLLMENT_GROUP_34_073: [This function shall save the provided capabilities.] */
```

Comment:
>SRS_ENROLLMENT_GROUP_34_073 [](start = 7, length = 27)

Do we still follow the Test_xxxx prefix here??

Reply:
I'll fix this, yeah